Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A basic ribbon control that houses <see cref="TabbedCommandBarItem"/>s
+    /// </summary>
+    [ContentProperty(Name = nameof(MenuItems))]
+    [TemplatePart(Name = "PART_RibbonNavigationView", Type = typeof(NavigationView))]
+    [TemplatePart(Name = "PART_RibbonContent", Type = typeof(ContentControl))]
+    [TemplatePart(Name = "PART_TabChangedStoryboard", Type = typeof(Storyboard))]
+    public class TabbedCommandBar : NavigationView
+    {
+        private ContentControl _ribbonContent = null;
+        private Storyboard _tabChangedStoryboard = null;
+
+        /// <summary>
+        /// Gets or sets the brush to use as the background for content
+        /// </summary>
+        public Brush ContentBackground
```

Comment:
Renamed to `ItemBackground`, but that's still not great. Maybe `RibbonBackground` would be better?

Reply:
We'd want to avoid using the term `Ribbon` anywhere, so `CommandBarBackground` would probably be better? Is it supposed to be the top bar or bottom bar's background color? I think that's what I was trying to understand.

Reply:
It's the background behind the CommandBar. I decided to just remove it and always have it set to the selected item's background. Maybe in the future it could be put back, but I just don't see a use for it (since it only appears briefly during the transition animation between tabs).